src/views/Sponsor/Prospectus.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <Tile bgcolor="bg-wri-teal" title="Why Sponsor WeRockIT?" titleColor="text-wri-dark-blue">
+    <div class="text-3xl pb-12 text-center text-white">
+      <p>
+        WeRockIT seeks to showcase diverse voices in technology while creating an inclusive environment that promotes
+        education, collaboration, and communication Our conference will offer panel discussions, educational
+        programming, networking opportunities, and hands-on workshops to expand technical and interpersonal skills.
+      </p>
+      <p>
+        WeRockIT Conf will run two tracks concurrently during the conference, a novice track and a senior track. We
+        anticipate 150-300 attendees from backgrounds such as Defense, Education, Cyber Security, Biotech, Space, and
+        many more.
+      </p>
+      <p>
+        By becoming involved in WeRockIT Conference, sponsors are supporting marginalized voices in tech in all stages
+        of their careers to achieve their potential as leaders and community stakeholders.
+      </p>
+    </div>
+    <div class="container mx-auto text-center">
+      <a
+        href="./WeRockIT2020SponsorshipProspectus.pdf"
+        target="_blank"
+        class="inline p-3 m-3 bg-purple-700 hover:bg-blue-800 rounded-full font-bebas text-white text-3xl"
+        >Download Prospectus</a
+      >
+      <a
+        href="mailto:contact@werockitconf.com?Subject=Sponorship%inquiry"
```

Comment:
```suggestion
        href="mailto:contact@werockitconf.com?Subject=Sponorship%20inquiry"
```

I think a space is %20